### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by lucy-sees**
- [x] Created an `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- [x] In the `main.rb` , I defined the entry point, this will be a method called `main` that is invoked at the end of your file. This method should do the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @lucy-sees, , it would be nice to remove the live demo section from your readme as you are not going to deploy this project, **please this is a comment not a review** 👍🏿

**Comment by lucy-sees** [comment]
@prolajumokeoni Kindly update review, I have made the suggested change

**Comment by prolajumokeoni** [comment]
Kindly await a review 👍🏿

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  when '6'
+    @app.list_rentals
+  else
+    exit
+  end
+end
+
+def main
+  @app = App.new
+  $stdout.clear_screen
```
Comment (prolajumokeoni) [inline main.rb]: - [ ] Good job so far, when I comment this line `$stdout.clear_screen`  and the code used to test Classes decorated in [person.rb](https://github.com/lucy-sees/Libra/blob/70995b8a64bc6c2694b201dd81b2e40cb119caaf/person.rb#L39) I can see the options presented to the user, Kindly make sure that your app works as expected  without errors

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    choice = gets.chomp
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    case choice
+    when '1'
+      print 'Has parent permission? [Y/N]: '
```
Comment (prolajumokeoni) [inline app.rb]: - [ ] Kindly make sure that a user can successfully create a student or teacher, I see this error on the console after this is fixed please run through all options presented to the user and ensure they work as expected, the user also cannot create a rental please check the rental method too 👍🏿 

person

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @lucy-sees 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Basic UI created 



## Required Changes ♻️

- [ ] Please fix the link to you MIT license page it is broken, when I click link in your readme it sows a 404 page  👍🏿 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (lucy-sees) [reply main.rb]: I have already made this change

Comment (prolajumokeoni) [reply main.rb]: - [ ] Hi @lucy-sees,  the part of the code I highlighted  in this review was green, I only saw your updates after I submitted my review 👍🏿

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    person = Teacher.new(age, specialization, name)
+    @all_persons.push(person)
+  end
+
+  def list_books
+    puts @all_books.inspect
+    @all_books.each do |book|
+      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
```
Comment (BenMukebo) [inline app.rb:60]: - [OPTONAL] There is some line above the list of books that you need to remove when listing the books option one (1). 👇🏻 


image

File: app.rb [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+      puts "#{index})[#{person_type}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+    person_index = gets.chomp.to_i
+    print 'Date: '
+    date = gets.chomp
+    Rental.new(date, @all_books[book_index], @all_persons[person_index])
+    puts 'Rental created successfully'
+  end
+
+  def get_rental(id)
```
Comment (BenMukebo) [inline app.rb:87]: - Great job 🎉 👍🏻 

image

**Review by BenMukebo — APPROVED** [review]
### Hi @lucy-sees, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- The school library console app behaves in the same was as the example video. 👍🏻 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**